server/form-pages/utils/defaultMatchingInformationValues.test.ts

```diff
@@ -49,11 +57,40 @@ describe('defaultMatchingInformationValues', () => {
     }
 
     expect(defaultMatchingInformationValues(body, assessment)).toEqual({
+      isCatered: 'essential',
       isWheelchairDesignated: 'essential',
       lengthOfStay: '24',
     })
   })
 
+  describe('isCatered', () => {
+    it('is set to the original value if defined', () => {
+      expect(
+        defaultMatchingInformationValues({ ...bodyWithUndefinedValues, isCatered: 'desirable' }, assessment),
+      ).toEqual(expect.objectContaining({ isCatered: 'desirable' }))
+    })
+
+    it("is set to 'essential' when there's no original value and `catering` === 'no'", () => {
```

Comment:
thanks for laying it out clearly in the commit message that 'catering' equals "self-catering" but maybe for extra clarity we should reinforce that potential gotcha in our scenario title e.g

> is set to 'essential' when there's no original value and `catering` (self-catering) === 'no'

Reply:
Good shout!